app/ui/src/app/dashboard/connections.component.ts

```diff
@@ -28,6 +30,12 @@ export class DashboardConnectionsComponent implements OnInit {
 
   truncateTrail = '…';
 
+  constructor(
+    private config: ConfigService
```

Comment:
Yeah, if we put this in the store then you don't need all these `ConfigService` injections etc.